ThreadGroup/Views/TGSelectDeviceStepView.m

```diff
@@ -65,6 +65,10 @@ - (void)commonInit {
 
 #pragma mark - Public
 
+- (void)setTopBarHidden:(BOOL)hidden {
+    self.topBar.hidden = hidden;
+}
```

Comment:
Okay, I'll change it! I was following the format of TGDeviceStepView, which does this.